plugins/drivers/proto/driver.proto

```diff
@@ -337,6 +337,9 @@ message TaskConfig {
 
     // JobName is the name of the job of which this task is part of
     string job_name = 13;
+
+    // Id of the associated allocation
```

Comment:
minor nitpick - the pattern wildly used in golang comments, and we seem to follow it here in protobuf (though not protobuf convention), is to start with the symbol name.

```suggestion
    // AllocId is the id of the associated allocation
```

Reply:
thanks, amended as such